src/main/java/org/mdbenefits/app/submission/conditions/ContactInfoMissing.java

```diff
@@ -0,0 +1,22 @@
+package org.mdbenefits.app.submission.conditions;
+
+import formflow.library.config.submission.Condition;
+import formflow.library.data.Submission;
+import java.util.Map;
+import org.springframework.stereotype.Component;
+
+/**
+ * Tests whether the record has a phone number or email address and if missing those, returns true;
+ */
+@Component
+public class ContactInfoMissing implements Condition {
+
+    @Override
+    public Boolean run(Submission submission) {
+        Map<String, Object> inputData = (Map) submission.getInputData();
+        String cellPhoneNumber = (String) inputData.get("cellPhoneNumber");
+        String emailAddress = (String) inputData.get("emailAddress");
+        return (cellPhoneNumber == null || cellPhoneNumber.isBlank()) &&
+                (emailAddress == null || emailAddress.isBlank());
+    }
```

Comment:
FYI, neither email or phone numbers are required.  The cellPhoneNumber format is already checked through field based validation logic, so it doesn't test it here. 

Basically, this only checks if they said text messages are okay and if so, ensures that a cell number is set.